cypress/integration/home_spec.js

```diff
@@ -3,3 +3,20 @@ describe('Home', () => {
     cy.visit('/')
   })
 })
+describe('PageNotFound', () => {
+  it('should load error page', () => {
+    cy.visit('/#/thisisnotavalidpath')
+  })
+})
+
+describe('People', () => {
+  it('should load', () => {
+    cy.visit('/#/people/1')
+  })
+})
+
+describe('People', () => {
```

Comment:
With BDD style you'd want to have unique `describe` blocks containing multiple `it` blocks if necessary. The `it` block below could join the other `it` block above.